control-plane/cmd/kafka-controller/main.go

```diff
@@ -63,5 +69,10 @@ func main() {
 		func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
 			return sink.NewController(ctx, watcher, sinkEnv)
 		},
+
+		// KafkaSource controller
+		func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
+			return source.NewController(ctx, watcher, sourceEnv)
+		},
```

Comment:
I have to add this since otherwise the data plane won't start

Reply:
funcy ....